Gemfile

```diff
@@ -14,3 +14,5 @@ if File.exist?(local_gemfile)
 else
   gem 'rails', '~> 5.0'
 end
+
+gem 'rubocop', git: 'git@github.com:bbatsov/rubocop.git'
```

Comment:
Don't think we need it here.

For local development you can use `Gemfile.local`.